controllers/helper.go

```diff
@@ -214,13 +190,15 @@ func syncPluginDaemonObjs(ctx context.Context,
 				return err
 			}
 			ds.Spec.Template.Spec.NodeSelector = dc.Spec.ConfigDaemonNodeSelector
+			// add the special node selector for the device plugin
+			ds.Spec.Template.Spec.NodeSelector[constants.SriovDevicePluginEnabledLabel] = constants.SriovDevicePluginEnabledLabelEnabled
```

Comment:
can't we put
```yaml
nodeSelector:
    sriovnetwork.openshift.io/device-plugin: Enabled
```
in `bindata/manifests/plugins/sriov-device-plugin.yaml` ?
should we take care of the ConfigDaemonNodeSelector?

Reply:
yep we must continue to run the device plugin only on nodes where we have the config daemon.

if I put it on the bindata it will get override no?

Reply:
I mean we might get rid of the ConfigDaemonNodeSelector info for the device plugin. The DrainController manages the `sriovnetwork.openshift.io/device-plugin: Enabled/Disabled` label only on nodes where the config daemon is running. Having the another selector is redundant, isn't it?

Reply:
ahhhh now I get your point! yes you are right!